examples/generic_builtin_bits.no

```diff
@@ -1,13 +1,23 @@
 use std::to_bits;
 use std::from_bits;
 
+// 010 = xx, where xx = 2
 fn main(pub xx: Field) {
+    // var
     let bits = to_bits(3, xx);
-    // 010 = xx, where xx = 2
     assert(!bits[0]);
     assert(bits[1]);
     assert(!bits[2]);
 
     let val = from_bits(bits);
     assert_eq(val, xx);
+
+    // constant
+    let cst_bits = to_bits(3, 2);
+    assert(!cst_bits[0]);
+    assert(cst_bits[1]);
+    assert(!cst_bits[2]);
```

Comment:
it'd be good to also `assert(cst_bits.len() == 3)`

thinking about that, I never liked that in circom you can write asserts that don't encode anything. So we may want to forbid `assert()` on a constant and have a `assert_dbg(cst_bits.len() == 3)` instead

Reply:
https://github.com/zksecurity/noname/issues/156